packages/react/src/components/form/initial.tsx

```diff
@@ -0,0 +1,25 @@
+import { InitialState } from "./flow";
+
+type Props = {
+  flowState: InitialState;
```

Comment:
@vincenzoiozzo This component can only be rendered in the `InitialState` so it expects the `flowState` to be of that type. No further checks are needed, eg in that state `.logIn` must be defined.